state/filesystem.go

```diff
@@ -1352,16 +1352,6 @@ func (sb *storageBackend) SetFilesystemAttachmentInfo(
 	if _, err := f.Info(); err != nil {
 		return errors.Trace(err)
 	}
-	// Also ensure the machine is provisioned.
```

Comment:
Is there code in the domain to replace this functionality? Or a Jira card if not